secret.tpl.json

```diff
@@ -1,8 +1,10 @@
-{
-  "gh_client_id": "${GH_CLIENT_ID}",
-  "gh_client_secret": "${GH_CLIENT_SECRET}",
-  "shieldsIps": [ "${SHIELDS_IP}" ],
-  "gh_token": "${GH_TOKEN}",
-  "npm_token": "${NPM_TOKEN}",
-  "wheelmap_token": "${WHEELMAP_TOKEN}"
-}
+{
+  "gh_client_id": "${GH_CLIENT_ID}",
+  "gh_client_secret": "${GH_CLIENT_SECRET}",
+  "shieldsIps": [ "${SHIELDS_IP}" ],
+  "gh_token": "${GH_TOKEN}",
+  "npm_token": "${NPM_TOKEN}",
+  "wheelmap_token": "${WHEELMAP_TOKEN}",
+  "bitbucket_server_username": "${BITBUCKET_SERVER_USERNAME}",
```

Comment:
I think we typically only add tokens here that are used by the main Shields.io service, and not self-hosted instances (see the [Jira service](https://github.com/badges/shields/blob/master/services/jira/jira-base.js#L14) for example. @paulmelnikow is that right?

Reply:
It's the opposite: this file is used in the Docker deployment, not in production.

Reply:
Gotcha, just wondering for the sake of consistency whether the BB auth info should be here, and/or if this should also be updated to include the auth info for all the other services (and if so if we should include this step in the docs as part of #2599)

Reply:
Definitely, I'd love to see the other options added there as well, it makes on-prem deployment a lot easier.

Reply:
Makes sense. I posted a note about that here: https://github.com/badges/shields/pull/2599#issuecomment-450378504. I'd like to fold the secrets and the environment into a single configuration method that covers both. Hopefully that could subsume this file and provide a consistent way that works for on-prem and PaaS without a deploy commit.

Reply:
Thinking about the conversation in https://github.com/badges/shields/pull/2599#discussion_r244350388
could we make these params `_user` and `_pass` as opposed to `_username` and `_password`.

Reply:
Sounds good, I'll rename them.